march_rqt_gait_selection/src/march_rqt_gait_selection/gait_selection_controller.py

```diff
@@ -0,0 +1,61 @@
+
+import ast
+import sys
+
+import rospy
+
+from march_shared_resources.srv import StringTrigger, Trigger
+
+
+class GaitSelectionController(object):
+    def __init__(self):
+        """Base class to communicate with the gait selection node."""
```

Comment:
Isn't this more a Class dosctring?

Reply:
I usually never put docstring on a class, always in the constructor